src/connectors/impls/http/server.rs

```diff
@@ -48,8 +48,7 @@ pub(crate) struct Config {
     tls: Option<TLSServerConfig>,
     /// custom codecs mapping from mime_type to custom codec name
     /// e.g. for handling `application/json` with the `binary` codec, if desired
-    #[serde(default)]
-    custom_codecs: HashMap<String, String>,
+    custom_codecs: Option<HashMap<String, String>>,
```

Comment:
```suggestion
#[serde(default)]
    mime_mapping: Option<HashMap<String, String>>,
```

Reply:
That's not needed, `Option` can be omitted without the need of default :)

Reply:
Ok, but the renaming is needed to keep it consistent.

Reply:
ohhhh I miseed that part 🤦 thanks!

Reply:
fixed in the current uncommited [art